src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const formatedDate = [];
+  const dateArr = date.split(fromFormat[fromFormat.length - 1]);
+  const dateObj = {
+    [fromFormat[0]]: dateArr[0],
+    [fromFormat[1]]: dateArr[1],
+    [fromFormat[2]]: dateArr[2],
```

Comment:
In this case, I see a hardcode with magic numbers ( index). 
You can try another method of filling the object.